app/models/disability_contention.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class DisabilityContention < ActiveRecord::Base
+  validates :code, presence: true, uniqueness: true
+  validates :medical_term, presence: true
```

Comment:
Is there a reason we're not validating `lay_term`?

Reply:
@egarreau Peggy confirmed that some lay_terms can be null